### DISCUSSION TITLE
refresh navigation badge

### DISCUSSION THREAD

**Discussion by panacz** [filamentphp/filament#7221] [Help] [answered]
Table polling works fine, but how can I update the corresponding navigation badge so that they are in sync?

**Comment by danharrin** [comment, marked as answer]
Not possible atm without reloading the page, the nav badge is outside of Livewire

**Reply by spizzo14** [reply]
@danharrin will this be possible soon? @heloufir FYI

**Reply by danharrin** [reply]
In v4, the sidebar content will be in its own Livewire component, so yes

**Reply by kingmaker-agm** [reply]
Thanks @danharrin, Will be eagerly waiting for these Features in the Filament v4.

Can you also make an Option for the Modals to be fired from Actions? 
Like making them separate modules as we did with the Filament Tables, Forms, Infolists in V3

**Reply by Merdin** [reply]
@danharrin 
> In v4, the sidebar content will be in its own Livewire component, so yes

How does this work now? Can't find a solution for it.

**Reply by leandrocfe** [reply]
https://filamentphp.com/docs/4.x/navigation/overview#reloading-the-sidebar-and-topbar

[example](https://x.com/leandrocfe/status/1887430297177702552)

**Reply by Merdin** [reply]
@leandrocfe 

Should it also work with the tabs of relation managers (tables)? Because that doesn't seem to work.

**Reply by leandrocfe** [reply]
Open a new help discussion and share your code please

**Reply by Merdin** [reply]
> Open a new help discussion and share your code please

https://github.com/filamentphp/filament/discussions/17760

**Comment by Arsham-rmz** [comment]
please add it to next update 💯

**Reply by cgoguyer** [reply]
Any news? an implementation date?

**Reply by slakbal** [reply]
Yeah while one is within the realm of Livewire there is a defaullt understanding that one should be able to listen within the components for some events from other livewire components. :) Have been working this problem with some folks on discord and then saw this discussion. Thanks to the folks assisting: 

Have a look at some examples that we tried without success, finally settled on a page refresh :)

https://discord.com/channels/883083792112300104/1269651024612823093
